spec/factories/users.rb

```diff
@@ -6,6 +6,7 @@
     duns_number { Faker::Company.duns_number }
     name { Faker::Name.name }
     email
+    github_login 'ghlogin'
```

Comment:
fwiw, anyone signing in via github will have a github login so adding the validation shouldn't be a problem